### PR TITLE
[WIP] FEATURE: Init branch variable

### PR DESCRIPTION
**Pull request by MalloZup**
the branch variable, will setup the run-script for running HEAD, MANAGER30, REFHOSTS

### CODE REVIEW

**Comment by MalloZup** [comment]
Ok tested, and worked. If branch="refhost", we run only the testhost. Otherwise we run manager30 branch by default if empty. This can be extended in future, if we have some other special GROUPS of test.

I think the future will be HEAD as default, and we will pass the manager30

File: main.tf.libvirt-testsuite.example [hunk 1]
```diff
@@ -52,7 +52,7 @@ module "control-node" {
   base_configuration = "${module.base.configuration}"
 
   name = "control-node"
-
+  # branch = "refhost" or keep it empty is manager30
```
Comment (moio) [inline main.tf.libvirt-testsuite.example]: Please remove this line but explain different options in README_ADVANCED.md

File: modules/libvirt/control_node/variables.tf [hunk 2]
```diff
@@ -28,6 +28,12 @@ variable "running" {
   default = true
 }
 
+variable "branch" {
+  description = "This variable set up the branch of spacewalk-testsuite, or the refhost"
```
Comment (moio) [inline modules/libvirt/control_node/variables.tf]: Please explicitly enumerate possible options here, eg. https://github.com/moio/sumaform/blob/master/modules/libvirt/client/variables.tf#L12

File: main.tf.libvirt-testsuite.example (hunk 1 above)
Comment (moio) [inline main.tf.libvirt-testsuite.example]: Please remove this line but explain different options in README_ADVANCED.md

File: modules/libvirt/control_node/variables.tf (hunk 2 above)
Comment (moio) [inline modules/libvirt/control_node/variables.tf]: Please explicitly enumerate possible options here, eg. https://github.com/moio/sumaform/blob/master/modules/libvirt/client/variables.tf#L12

**Comment by MalloZup** [comment]
So, @moio, needed changes are fixed now. I let this Pr in WIP status, because this depend strictly for this other PR, that i'm working on.
https://github.com/moio/sumaform/pull/22

**Comment by MalloZup** [comment]
@moio, i'm closing, since priority for moment is only 3.0

**Pull request closed without merging**